Mage.Client/src/main/java/mage/client/components/HoverButton.java

```diff
@@ -45,7 +47,7 @@ public class HoverButton extends JPanel implements MouseListener {
 
     private String topText;
     private Image topTextImage;
-    private Image topTextImageRight;
+    private ArrayList<Image> topTextImagesRight = new ArrayList<>();
```

Comment:
Can be final?
Declare field as interface `List<Image>` rather than implementation ArrayList

Reply:
Sure, out of curiosity, what difference does `List` vs `ArrayList` make?